mbenfowler/sound-stash
# Feature/palette selector

## What I did:
Added dark and light mode!

Remember to `npm install` because I added a package for the light/dark toggle component

## Testing Notes/Extras: 

1. Ensure all user flows are functioning as expected
2. All e2e tests pass
3. Toggle correctly toggles between light and dark mode.  It's position looks good no matter what size screen you're on.
4. Observe where all the "light" mode colors are being consumed. Do these colors work? Should any other colors be used instead?

Known issues:
1. With variables I am unable to affect the `background-color` we have set on our `html` tag in App.css. It doesn't look terrible, but it is somewhat noticeable especially on light mode.
2. ~~Journal icons are white which makes them invisible in our light mode.  Will need to add more custom icons to fix.~~ I used filter to solve for this
